### DISCUSSION TITLE
password-management-en/

### DISCUSSION THREAD

**Discussion by giscus** [iliayatsenko/iliayatsenko.github.io#24] [Announcements]
# password-management-en/

Ilia's notes about software architecture, testability, and other, more mundane things

https://iliayatsenko.github.io/password-management-en/

**Comment by danyaYatsik** [comment]
Brilliant!! That's what I've needed half of my whole life

**Reply by iliayatsenko** [reply]
Yes, this utility is really cool, I enjoy using it. Despite it requires a little bit more steps to grab password comparing to password managers, the feeling that (almost) everything is under your control is invaluable)
